packages/block-library/src/overlay-close/block.json

```diff
@@ -0,0 +1,57 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
```

Comment:
Should this have a `"parent"` definition to restrict insertion to the overlay template part?